src/validator/CreditCardVendorReading.java

```diff
@@ -0,0 +1,7 @@
+package validator;
+
+import java.util.ArrayList;
+
+public interface CreditCardVendorReading {
+	public ArrayList<CreditCardVendor> returnVendorArray();
```

Comment:
Declarations should use Java collection interfaces such as "List" rather than specific implementation classes such as "ArrayList"

Method name returnVendorArray reveals implementation details, should be getVendorList or better getVendors.

Modifier public is redundant for interface methods.

The name does not follow the class convention. It should not be CreditCardVendorReading, but CreditCardVendorReadable or CreditCardVendorReader (we already have that class). In general, this interface is implemented only by CreditCardVendorsReader.  Do we expect a second CreditCardVendorsReader implementation? If no, we can remove this interface.